### PR TITLE
Don't send return-only events to poll

### PR DESCRIPTION
**Pull request by Parlane**
Fixes some network issues where the game/system asked for POLLHUP specifically.

### CODE REVIEW

File: Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp [hunk 1]
```diff
@@ -980,173 +980,179 @@ IPCCommandResult CWII_IPC_HLE_Device_net_ip_top::IOCtl(u32 _CommandAddress)
...
-				int unhandled_events = events;
-				ufds[i].events = 0;
-				for (auto& map : mapping)
-				{
-					if (events & map[1])
-						ufds[i].events |= map[0];
-					unhandled_events &= ~map[1];
-				}
+			// Do not pass return-only events to the native poll
+			ufds[i].events &= ~(POLLHUP | POLLHUP | POLLNVAL);
```
Comment (delroth) [inline Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp]: POLLHUP twice?

Comment (Juesto) [inline Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp]: ><FireFly> Juesto: I'd think the double POLLHUP is a copy-paste error, and one of them was meant to be changed into another signal

Comment (Parlane) [inline Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp]: Yep pretty dumb mistake. Was meant to be POLLERR

**Comment by JMC47** [comment]
lgtm

**Comment by lioncash** [comment]
LGTM

**Comment by Juesto** [comment]
Thanks for merging/fixing :+1:

**Pull request merged**